tests/test_core.py

```diff
@@ -8425,6 +8431,7 @@ def setUp(self):
 
 # Experimental modes (not tested by CI)
 lld = make_run('lld', emcc_args=[], settings={'LLD_REPORT_UNDEFINED': 1})
+minimal0 = make_run('minimal', emcc_args=['-g'], settings={'MINIMAL_RUNTIME': 1})
```

Comment:
why `-g`?

Reply:
Just so that failures have functions names in the backtraces.

Reply:
`--profiling` would also support that and would be faster to run (no DWARF). but doesn't matter much if not run by default.